code/game/machinery/pipe/construction.dm

```diff
@@ -137,20 +149,39 @@ Buildable meters
 
 	var/obj/machinery/atmospherics/fakeA = pipe_type
 	var/flags = initial(fakeA.pipe_flags)
+	var/pipe_count = 0
+	for(var/obj/machinery/atmospherics/M in loc)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
done, next commit